scripts/breakout-to-secureCRT-session/main.py

```diff
@@ -4,16 +4,14 @@
 requests.packages.urllib3.disable_warnings()
 from cmlApiCalls import CML as cml
 
-#edit the following variables
-server = "cml.server.com"
+server = "cml.recurve-it.com"
 username = "admin"
-password = "CMLpassword123"
+password = "12qwas!@QWAS12qwas"
```

Comment:
Do you really want to put a password in here?

Reply:
ahh crap, forgot to remove my lab password when i tested lol. 
good catch.